server/service/src/sync/translations/purchase_order_line.rs

```diff
@@ -0,0 +1,247 @@
+use chrono::NaiveDate;
+use repository::{
+    ChangelogRow, ChangelogTableName, PurchaseOrderLineRow, PurchaseOrderLineRowRepository,
+    StorageConnection, SyncBufferRow,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::sync::{
+    sync_serde::{date_option_to_isostring, empty_str_as_option, zero_date_as_option},
+    translations::{
+        master_list::MasterListTranslation, name::NameTranslation, period::PeriodTranslation,
+        purchase_order::PurchaseOrderTranslation, store::StoreTranslation, PullTranslateResult,
+        PushTranslateResult, SyncTranslation,
+    },
+};
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Serialize, Clone, Debug)]
+pub struct LegacyPurchaseOrderLineRow {
```

Comment:
Missing `status`?

Reply:
Status is added - I think we both missed it in hidden large purchase_order dif